core/src/main/resources/hudson/model/Node/help-labelString.html

```diff
@@ -1,11 +1,22 @@
 <div>
-  Labels (AKA tags) are used for grouping multiple agents into one logical group.
-  Use spaces between each label. For instance 'regression java6' will assign a
-  node the labels 'regression' and 'java6'.
-
-  <p>
-  For example, if you have multiple Windows agents and you have jobs that require
-  Windows, then you can configure all your Windows agents to have the label 'windows',
-  then tie the job to the 'windows' label. This allows your job to run on any
-  of your Windows agents but not on anywhere else. 
-</div>
\ No newline at end of file
+  Labels (or tags) are used to group multiple agents into one logical group.
+  <p>
+  For example, if you have multiple Windows agents and you have a job that
+  must run on Windows, then you can configure all your Windows agents to have
+  the label <tt>windows</tt>, and then tie that job to this label.
+  <br>
+  This would ensure that your job runs on one of your Windows agents, but not
```

Comment:
Yeah, sure. Tenses are still a bit mixed up here anyway.

Also, I wonder about:
- *your job runs* on of your agents
- *the job gets built* on one of your agents
- *builds of this job* run on one of your agents

etc.